pages/analysis-steps/[id].js

```diff
@@ -0,0 +1,146 @@
+// node_modules
+import Link from "next/link";
+import PropTypes from "prop-types";
+// components
+import Attribution from "../../components/attribution";
+import Breadcrumbs from "../../components/breadcrumbs";
+import {
+  DataArea,
+  DataAreaTitle,
+  DataItemLabel,
+  DataItemValue,
+  DataPanel,
+} from "../../components/data-area";
+import DocumentTable from "../../components/document-table";
+import { EditableItem } from "../../components/edit";
+import JsonDisplay from "../../components/json-display";
+import ObjectPageHeader from "../../components/object-page-header";
+import PagePreamble from "../../components/page-preamble";
+// lib
+import buildBreadcrumbs from "../../lib/breadcrumbs";
+import { requestDocuments } from "../../lib/common-requests";
+import errorObjectToProps from "../../lib/errors";
+import FetchRequest from "../../lib/fetch-request";
+import AliasList from "../../components/alias-list";
+import buildAttribution from "../../lib/attribution";
+import { isJsonFormat } from "../../lib/query-utils";
+
+export default function AnalysisStep({
+  analysisStep,
+  documents,
+  attribution = null,
+  isJson,
+}) {
+  return (
+    <>
+      <Breadcrumbs />
+      <EditableItem item={analysisStep}>
+        <PagePreamble />
+        <ObjectPageHeader item={analysisStep} isJsonFormat={isJson} />
+        <JsonDisplay item={analysisStep} isJsonFormat={isJson}>
+          <DataPanel>
+            <DataArea>
+              <DataItemLabel>Title</DataItemLabel>
+              <DataItemValue>{analysisStep.title}</DataItemValue>
+              {analysisStep.analysis_step_types?.length > 0 && (
+                <>
+                  <DataItemLabel>Analysis Step Types</DataItemLabel>
+                  <DataItemValue>
+                    {analysisStep.analysis_step_types.join(", ")}
+                  </DataItemValue>
+                </>
+              )}
+              {analysisStep.workflow && (
+                <>
+                  <DataItemLabel>Workflow</DataItemLabel>
+                  <DataItemValue>
+                    <Link href={analysisStep.workflow["@id"]}>
+                      {analysisStep.workflow.accession}
+                    </Link>
+                  </DataItemValue>
+                </>
+              )}
+              {analysisStep.input_content_types?.length > 0 && (
```

Comment:
`input_content_types` is required, though it doesn’t have a `minItems`, so you can remove the `?`.